lib/Unexpected.js

```diff
@@ -898,15 +910,14 @@ Unexpected.prototype.throwAssertionNotFoundError = function (subject, testDescri
     });
 };
 
-Unexpected.prototype.lookupAssertionRule = function (subject, testDescriptionString, args) {
+Unexpected.prototype.lookupAssertionRule = function (subject, testDescriptionString, args, requireAssertionSuffix) {
     var that = this;
     if (typeof testDescriptionString !== 'string') {
         throw new Error('The expect function requires the second parameter to be a string.');
     }
-
     var handlers = this.assertions[testDescriptionString];
     if (!handlers) {
-        this.throwAssertionNotFoundError(subject, testDescriptionString, args);
+        return;
```

Comment:
I would prefer that, but I don't care deeply about it. I know you don't like that JavaScript has two values for nothing. I don't either but I tend use `null` as intentional undefined. It is so easy ending up with `undefined` by mistake, that does not happen with `null`.